index.html

```diff
@@ -0,0 +1,144 @@
+</head>
+
+<body>
+
+    <!--Navbar-->
+    <div class="navbar">
+        <div class="navbar__logo-links">
+            <h3 class="navbar__log">Medical Core</h3>
+            <ul class="navbar__links">
+                <li class="link"><a href="#"></a>Home</li>
+                <li class="link"><a href="#"></a>Product</li>
+                <li class="link"><a href="#"></a>Pricing</li>
+                <li class="link"><a href="#"></a>Contact</li>
+            </ul>
+        </div>
+        <div class="navbar__button">
+            <ul class="navbar__links">
+                <a class="link link__text" href="#">Login</a>
+                <a class="btn" href="#">JOIN US &nbsp; <i class="fa-solid fa-arrow-right"></i></a></li>
+            </ul>
+        </div>
+    </div>
+
+
+    <!--Header-->
+    <header class="header">
+        <div class="header-content">
+            <h5 class="name-message">Welcome</h5>
+            <h1 class="header__head">A Great Place to <br> Receive Care</h1>
+            <p class="header__desc">Get your best looking smile now!</p>
+            <div class="link link-header_btn">
+                <a href="#" class="btn">Join Us</a>
+                <a href="#" class="btn btn--outlined">Learn More</a>
+            </div>
+        </div>
+    </header>
+
+<!--Main-->
+    <main class="main">
+        <div class="main__img"><img src="/img/thumb-concept.svg" alt="Main doctor"></div>
+        <div class="main__content">
+            <span class="main__line"></span>
+            <h2 class="title">Meet Our Experts</h2>
+            <p class="text">Problems trying to resolve the conflict between <br>
+                the two major realms of Classical physics: <br>
+                Newtonian mechanics </p>
+            <a class="link link--highlighted" href="#">Learn More <i class="fa-solid fa-chevron-right"></i></a>
+        </div>
+    </main>
+
+    <!--Doctors Section-->
+    <div class="container">
+        <div class="container__content">
+            <p class="descr">Team</p>
+            <h5 class="title title__style">Our Team</h5>
+            <p class="text">Problems trying to resolve the conflict between <br>
+                the two major realms of Classical physics: Newtonian mechanics </p>
+        </div>
+        <div class="card">
+            <div class="container__card">
+                <img class="card__img" src="/img/user-cover-1.png" alt="">
+                <div class="card__body">
+                    <h5 class="name-message name-message--style">Julian Jameson</h5>
+                    <p class="card__prof">Profession</p>
+                </div>
+                <div class="card__footer">
+                    <a href="#"><i class="fa-brands fa-facebook"></i></a>
+                    <a href="#"><i class="fa-brands fa-instagram"></i></a>
+                    <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                </div>
+            </div>
+
+            <div class="container__card">
```

Comment:
`container__card` this is not bem find. you're using the element outside its block